### ISSUE TITLE
YAML module is not available.

### ISSUE THREAD

**Issue by SuperShadowPlay** [neocl/freebible#1]
When importing the `bible` function I get this error. Any ideas why? I'm running python3.6 on Ubuntu 16.04

```>>> from freebible import bible
YAML module is not available. Exporting to YAML function will NOT work.
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/.local/lib/python3.6/site-packages/freebible/__init__.py", line 15, in <module>
    from freebible.helpers import read_kougo, read_web, bibles
  File "/home/user/.local/lib/python3.6/site-packages/freebible/helpers.py", line 16, in <module>
    from freebible.parsers.kougo import parse_kougo
  File "/home/user/.local/lib/python3.6/site-packages/freebible/parsers/kougo.py", line 38, in <module>
    from chirptext.io import read_file
ModuleNotFoundError: No module named 'chirptext.io'```

**Comment by letuananh** [comment]
Hi, sorry I missed this comment. This bug was caused by an update in https://github.com/letuananh/chirptext module (renaming chirptext.io to chirptext.chio). It was fixed in the latest release (freebible-0.1a8). Can you please kindly help to update chirptext and freebible packages and see if it is working as intended? Thank you for contributing.

```bash
pip install --upgrade chirptext
pip install --upgrade freebible
```

**Comment by SuperShadowPlay** [comment]
It worked, thanks!

```
>>> from freebible import bibles

>>> bibles.print("Exo", 5, 20)

[Exo 5:20] 彼らパロをはなれて出たる時モーセとアロンの對面にたてるを見たれば 
[Exo 5:20] They met Moses and Aaron, who stood in the way, as they came forth from Pharaoh:
[Exo 5:20 彼らパロをはなれて出たる時モーセとアロンの對面にたてるを見たれば , Exo 5:20 They met Moses and Aaron, who stood in the way, as they came forth from Pharaoh:]
